lib/grammers-client/src/types/inline_query.rs

```diff
@@ -0,0 +1,196 @@
+use super::{Chat, ChatMap};
+use crate::{client::Client, utils::generate_random_id, InputMessage};
+use grammers_mtsender::InvocationError;
+use grammers_tl_types as tl;
+use std::sync::Arc;
+
+/// Represents an inline query update, which occurs when you sign in as a bot and a user sends an
+/// inline query such as `@bot query`.
+pub struct InlineQuery {
+    query: tl::types::UpdateBotInlineQuery,
+    client: Client,
+    chats: Arc<ChatMap>,
+}
+
+/// An inline query answer builder.
+pub struct Answer {
+    request: tl::functions::messages::SetInlineBotResults,
+    client: Client,
+}
+
+/// An inline query result.
+///
+/// grammers currently supports results of the following types that implement [`Into<InlineResult>`]:
```

Comment:
I don't recall other doc-comments mentioning the name of the library, so it sounds a bit weird.

```suggestion
/// The following types that implement [`Into<InlineResult>`]:
```